torch/fx/experimental/proxy_tensor.py

```diff
@@ -211,11 +210,7 @@ def get_constant(idx):
 def maybe_disable_fake_tensor_mode():
     # TODO: figure out if this API generally makes sense and bake it into the
     # library
-    mb_fake_mode = _get_current_dispatch_mode()
-    if isinstance(mb_fake_mode, FakeTensorMode):
-        return _pop_mode_temporarily()
-    else:
-        return nullcontext()
+    return unset_fake_temporarily()
```

Comment:
you can just get rid of this function entirely now right :)

Reply:
I was a liiiitle worried that this is used externally (since I see it used a bunch in the codebase), so I figured I'd save for a followup :)